src/components/campaign/campaignInfo.jsx

```diff
@@ -49,8 +50,8 @@ const CampaignInfo = props => {
         <div>
           Ward Concentration Score: {_.round(info.wardConcentrationScore, 5)}
         </div>
-      </div>
-      <div className="col-sm-12">
+      </Col>
+      <Col sm={12}>
```

Comment:
I didn't check, it looked odd since we're using react-bootstrap everywhere else.